<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>36</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_state">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>36</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_man">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_women">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>35</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_bang">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>棒棒哒</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_meng">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>萌萌哒</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_yao">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>该吃药了</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>年龄段</string>
   </property>
   <widget class="QRadioButton" name="radioButton_0to19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0-19</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_20to39">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>20-39</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_40to59">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>40-59</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_50up">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>50以上</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
